kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/ConvertUtils.java

```diff
@@ -577,6 +577,7 @@ public static ProcessInstanceUserTaskWithVariablesList convertToUserTaskWithVari
             ProcessInstanceUserTaskWithVariables var = new ProcessInstanceUserTaskWithVariables();
             var.setId(desc.getTaskId());
             var.setName(desc.getName());
+            var.setDescription(desc.getDescription());
```

Comment:
Same as above.
You should apply the same to the `ConvertUtils` class in the case-mgmt package.

https://github.com/kiegroup/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-services/kie-server-services-case-mgmt/src/main/java/org/kie/server/services/casemgmt/ConvertUtils.java